README.md

```diff
@@ -18,48 +17,47 @@ This operator is based on [kubebuilder SDK](https://book.kubebuilder.io/).
 - [Operator Downgrade](#downgrade)
 - [Operator Removal](#removal)
 
-## Why use the Wavefront Operator for Kubernetes?
+## Why Use the Observability for Kubernetes Operator?
 
-The operator simplifies operational aspects of managing the Wavefront integration. Here are some examples, with more to come!
- - Enhanced status reporting of the Wavefront integration so that users can ensure their cluster and Kubernetes resources are reporting data.
- - Kubernetes Operator features provide a declarative mechanism for deploying the Wavefront Collector and proxy in a Kubernetes environment.
+The Operator simplifies operational aspects of managing the Kubernetes Integration for VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront). Here are some examples, with more to come!
+ - Enhanced status reporting of the Kubernetes Integration so that users can ensure their cluster and Kubernetes resources are reporting data.
+ - Kubernetes Operator features provide a declarative mechanism for deploying the Kubernetes Metrics Collector and Wavefront proxy in a Kubernetes environment.
  - Centralized configuration.
  - Enhanced configuration validation to surface what needs to be corrected in order to deploy successfully.
  - Efficient Kubernetes resource usage supports scaling  out the cluster (leader) node and worker nodes independently.
 
-**Note:** The Collector that is deployed by the Operator still supports configuration via configmap.
+**Note:** The Kubernetes Metrics Collector that is deployed by this Operator still supports configuration via configmap.
 For example, Istio and MySQL metrics, Telegraf configuration, etc. are still supported. For details on the Collector, see [collector.md](docs/collector/collector.md).
 
 ## Architecture
 
-![Wavefront Operator for Kubernetes Architecture](docs/images/architecture-logging.png)
+![Observability for Kubernetes Operator Architecture](docs/images/architecture-logging.jpg)
 
 # Installation
 
-**Note:**  The Wavefront Operator for Kubernetes helm chart has been deprecated and is no longer supported. Use the deploy, upgrade and removal instructions below instead.
+**Note:**  The Observability for Kubernetes Operator Helm chart is deprecated and no longer supported. Use the deploy, upgrade, and removal instructions below instead.
 
 ## Prerequisites
 
-The following tools are required for installing the integration.
-- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+To install the integration, you must use the [kubectl](https://kubernetes.io/docs/tasks/tools/) tool.
 
 
-## Deploy the Wavefront Collector and Proxy with the Operator
+## Deploy the Kubernetes Metrics Collector and Wavefront Proxy with the Observability for Kubernetes Operator
```

Comment:
Hmm. How about Install Aria Kubernetes agents with

Reply:
we could simplify it down to like "monitoring agents" or something, aim for semantics over branding